src/css.js

```diff
@@ -263,8 +264,12 @@ jQuery.extend( {
 			// If a hook was provided, use that value, otherwise just set the specified value
 			if ( !hooks || !( "set" in hooks ) ||
 				( value = hooks.set( elem, value, extra ) ) !== undefined ) {
+				if ( typeof value === "string" && value.includes( "!important" ) ) {
 
-				if ( isCustomProp ) {
+					// Support !important priority (gh-3713).
+					var splitValue = value.split( "!" );
+					style.setProperty( argName,  splitValue[ 0 ], splitValue[ 1 ] );
+				} else if ( isCustomProp ) {
```

Comment:
Is it valid to say that custom properties never use `!mportant`? I would sure hope you can't override CSS variables with it or some other crazy thing 😵 but I'll ask anyway.

Reply:
CSS vars do support “!important”. They’re basically like any other CSS prop, just with some additional features, the most important one that they’re not dropped by the engine what happens with regular unrecognized CSS props.